src/octoprint/static/js/app/viewmodels/files.js

```diff
@@ -423,6 +423,17 @@ $(function() {
                     }
                 }
                 output += gettext("Estimated Print Time") + ": " + formatDuration(data["gcodeAnalysis"]["estimatedPrintTime"]) + "<br>";
+		if(self.loginState.isUser()&&data["gcodeAnalysis"]["warning"])
+		{
+		    var warning = "<p>" + gettext("Object exceeds printing area") + "</p>";
+                    warning += pnotifyAdditionalInfo("<pre>Object or objects in file '"+data["name"]+"' exceed/s the printing area</pre>");
```

Comment:
Do we really need the additional information here? Does it really add anything? The file name could be included in the initial message (use `sprintf` so that the translation is still generic and only contains a placeholder for the filename, there are examples throughout the code on how to do that), and the rest of the message is just repetition really. It might make sense to include information which axes exceed the build volume (x, y or z, or maybe more than one?).